cmd/lakefs/cmd/run.go

```diff
@@ -183,6 +183,7 @@ var runCmd = &cobra.Command{
 			interval := cfg.GetSecurityAuditCheckInterval() + jitter
 			auditChecker.StartPeriodicCheck(ctx, interval, logger)
 		}
+		// Wiring required for sending emails once it's decided where exactly it will be call from
```

Comment:
what about this part? not implementing yet?
If so, leave it as a TODO (`// TODO(shimi): `)